samples/bluetooth/hci_open_amp/src/main.c

```diff
@@ -0,0 +1,469 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <stddef.h>
+#include <stdio.h>
+#include <string.h>
+
+#include <zephyr.h>
+#include <arch/cpu.h>
+#include <sys/byteorder.h>
+#include <logging/log.h>
+#include <sys/util.h>
+#include <ipm.h>
+
+#include <openamp/open_amp.h>
+#include <metal/sys.h>
+#include <metal/device.h>
+#include <metal/alloc.h>
+
+#include <net/buf.h>
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/l2cap.h>
+#include <bluetooth/hci.h>
+#include <bluetooth/buf.h>
+#include <bluetooth/hci_raw.h>
+
+/* Configuration defines */
+
+#define LOG_LEVEL LOG_LEVEL_INFO
+#define LOG_MODULE_NAME hci_open_amp
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+#define SHM_START_ADDR      (0x20010000 + 0x400)
```

Comment:
Is there a DTS symbol for shared memory?

Reply:
@galak ^^

Reply:
Yes, @kapi-no there is DTS information in the nrf5340 DTS structure that points to the shared memory.

Reply:
Yes, I see - there is ```sram0_shared``` DT node, but I don't see any alias / define generated for it